taxi/views.py

```diff
@@ -44,6 +65,26 @@
     model = Car
 
 
+class CarCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Car
+    fields = "__all__"
```

Comment:
Using `fields = "__all__"` in `CreateView` exposes all fields of the `Car` model in the form. Ensure that this is intentional and that no sensitive fields are included. Consider specifying only the fields you want to expose.